RELEASE.md

```diff
@@ -133,18 +133,17 @@ This repo includes a "development" chart in the [charts/](./charts/fluent-operat
 
 Fluent Operator uses [custom builds](./cmd/fluent-watcher/README.md) of both Fluentd and Fluent-bit.  These images can (and often should be) be published out-of-band of Fluent Operator releases.
 
-## Fluent-bit 
+## Fluent-bit
 
 To publish a new fluent-bit image:
 
-* Execute the [bump-fluent-bit-version](https://github.com/fluent/fluent-operator/actions/workflows/bump-fluent-bit-version.yaml) workflow dispatch to generate a PR to update fluent-bit version references in this repo 
-* Merge the PR generated by the [bump-fluent-bit-version](https://github.com/fluent/fluent-operator/actions/workflows/bump-fluent-bit-version.yaml) workflow
-* Execute the [build-fb-image](https://github.com/fluent/fluent-operator/actions/workflows/build-fb-image.yaml) workflow dispatch to build and publish the new image 
+* Execute the [bump-fluent-bit-version](https://github.com/fluent/fluent-operator/actions/workflows/bump-fluent-bit-version.yaml) workflow dispatch to generate a PR to update fluent-bit version references in this repo
```

Comment:
This would change once the renovate PR is created. Basically the step would be removed.